tst/testinstall/xgap.tst

```diff
@@ -53,10 +51,6 @@ gap> HasSize(t);
 true
 gap> Size(t); 
 1
-gap> f := FreeGroup(2);;
-gap> g:=f/[f.1^2,f.2^3];;
-gap> g.1^5=g.1;
-true
 gap> STOP_TEST( "xgap.tst", 62690000);
```

Comment:
It took several seconds (5-10?) to complete, so does not belong into `testinstall`. 

I contemplated moving it to a new test file in `teststandard`, but ultimately decided that this is a weird random test, for whch it is unclear what exactly it is meant to exercise. Actually, the whole `xgap.tst` somewhat falls into that category. From the original commit by Max N., who added it 1999-05-30: "Added tests for fp groups for problems found with XGAP. Max."

So ultimately, this file should go, its contents (or equivalent content) distributed over more appropriate files (e.g. `fpgrps.tst` or files in `opers`).